armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -257,13 +256,21 @@ After=network.target startup-checks.service
 Requires=startup-checks.service
 [Service]
 ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf 
+# Note: we need this ExecStartPost command to prepare an .env file
+# in expected format for electrs to have the RPCPASSWORD in its
+# environment. If we contributed a --cookie-file flag to upstream
+# electrs that could parse the .cookie file, we would not need this.
+ExecStartPost=/bin/bash -c " \
+  sleep 5 && \
+  echo -n "RPCPASSWORD=" > /mnt/ssd/bitcoin/.bitcoin/.cookie.env && \
+  tail -c +12 /mnt/bitcoin/.bitcoin/.cookie >> /mnt/ssd/bitcoin/.bitcoin/.cookie.env"
```

Comment:
> `tail -c +12 /mnt/bitcoin/.bitcoin/.cookie`

typo: should be `tail -c +12 /mnt/ssd/bitcoin/.bitcoin/.cookie`

Reply:
Oops, good catch.

Thanks, fixed.